docs/variants/novacustom_ns5x_adl/overview.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+
+## Hardware
+
+The `NovaCustom NS5xPU/NS7xPU` section is related to the two physical
+NovaCustom laptops based on the 12th generation Intel Core processor:
+
+- [NS51 series](https://configurelaptop.eu/ns51-series/)
+- [NS70 series](https://configurelaptop.eu/ns70-series/)
+
+![](images/NS51-front-1.png)
```

Comment:
I wonder why we don't have picture with Dasharo logo

Reply:
Because we have not taken these photos ourselves. Me might do that in next iteration.